types/react-syntax-highlighter/index.d.ts

```diff
@@ -53,6 +56,7 @@ declare module 'react-syntax-highlighter/dist/esm/light' {
     import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
     export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {
         static registerLanguage(name: string, func: any): void;
+        static supportedLanguages: string[];
```

Comment:
It doesn't appear this property exists (is added) to the light version: https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/v13.5.3/src/light.js

I've verified this against the package installed in my project.